samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>First we will create the model using the&nbsp;<code>EntityRecognizer()</code>&nbsp;constructor and passing the following parameters:</p><p><code>data</code>: The databunch created using the&nbsp;<code>prepare_textdata</code>&nbsp;method.</p><p><code>backbone</code>: To use mistral as the model backbone, use&nbsp;<code>backbone="mistral"</code>.</p><p><code>prompt</code>: Text string describing the task and its guardrails. This is an optional parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>